src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -47,23 +46,23 @@ While _written language_ and _character_ are slightly less precise, they should
 If you must use a word that your audience is unlikely to know, provide a definition using the [Key-term Aside component](/handbook/web-dev-components#asides).
 
 ## Make information easy to find
-Headings [shouldn't skip levels](/heading-levels). Also, don't add custom styles to headings—it makes the page hierarchy more difficult to follow for sighted readers.
+[Don't skip heading levels](/heading-levels). Don't add custom styles to headings—it makes the page hierarchy more difficult to follow.
 
 Spell out acronyms the first time they're used. For example: _Web Incubation Community Groups (WICG)_.
 
-Exceptions, edge cases, and other kinds of supplemental information should be right next to the primary content they're related to.
+Place exceptions, edge cases, and other kinds of supplemental information right next to the primary content they're related to.
 
 ## Use inclusive images
 Always provide [alt text](/image-alt).
 
 If you're creating an illustration, the parts that are essential for understanding the illustration should have a contrast ratio of at least 3:1. You can verify the contrast using checkers like the ones from [WeAIM](https://webaim.org/resources/contrastchecker/) or the [Paciello Group](https://developer.paciellogroup.com/resources/contrastanalyser/).
 
-Avoid images that may exclude certain audience members. Just like when we _write_ about people, we must remain inclusive when we _show_ people and the things they do and make. For example, avoid stock photos that show only male developers.
+Avoid images that may exclude certain audience members. Just like when you _write_ about people, remember to be inclusive when you _show_ people and the things they do and make. For example, avoid stock photos that show only male developers.
 
 ## Create accessible code blocks
-While code blocks should be as [simple as possible](/handbook/style#keep-it-simple), they must always include accessibility best practices. At a minimum:
-* Form elements should have [labels](/labels-and-text-alternatives/#label-form-elements).
-* Images should have [alt text](/image-alt).
+Make sure code blocks are [simple](/handbook/style#keep-it-simple) and follow  accessibility best practices. At a minimum:
```

Comment:
This changes the meaning--I wanted to convey that accessibility details are an exception to the general goal of keeping sample code simple.